### ISSUE TITLE
Sonarqube doesn't seems to be maintain anymore

### ISSUE THREAD

**Issue by btrd** [Scalingo/documentation#1485]
https://doc.scalingo.com/platform/getting-started/getting-started-with-sonarqube

This page is talking about something we don't maintain anymore. Not sure we should keep it.

**Comment by EtienneM** [comment]
I agree that SonarQube support should be dropped 'or announced as not maintained anymore). But I don't think it should only be about updating the documentation. Deprecating a supported technology takes a bit more actions than removing the documentation :)
